Cargo.toml

```diff
@@ -20,6 +20,9 @@ wasm-client = ["js-sys", "web-sys", "wasm-bindgen", "wasm-bindgen-futures"]
 middleware-logger = []
 
 [dependencies]
+async-compression = {version = "0.1.0-alpha.1"}
+bytes = "0.4.12"
+accept-encoding = "0.2.0-alpha.2"
```

Comment:
We probably should make the dependencies part of a `middleware-compression` feature.